### ISSUE TITLE
Why does android application is not supported for Xiomi Redmi Y2 mobile phone

### ISSUE THREAD

**Issue by rebelgiri** [corona-warn-app/cwa-app-android#993]
## Avoid duplicates
* [ ] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Question is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Question has not already been asked in another issue

## Your Question

* Source File:
* Line(s):
* Question:

**Comment by daimpi** [comment]
Which Android version are you using and do you have Google Play services installed?

**Comment by rebelgiri** [comment]
Google Play services, Yes I have installed.
Android version 9 PKQ1.181203.001

**Comment by rebelgiri** [comment]
I just realized this application is not available worldwide. It is only for Germany. So I can not download It.
Actually I wanted to scan the barcode given by corona test center to determine the results of covid-19 test.

**Comment by GPclips** [comment]
Hello @rebelgiri,

the Corona-Warn app currently is available in multiple European countries. Please check out the FAQ: https://www.coronawarn.app/en/faq/#international.
You should be able to switch to one of these PlayStores to be able to download the application.

Thanks,
LMM

Corona-Warn-App Open Source Team

**Comment by rebelgiri** [comment]
The problem is Google Play Store allows to change region only once in a year.

**Comment by daimpi** [comment]
@rebelgiri which country are you currently using? 
If its not yet supported, you can create an entry in the wishlist repo for your country or upvote an existing one: https://github.com/corona-warn-app/cwa-backlog/issues/9#issuecomment-648745301

**Comment by kbobrowski** [comment]
@rebelgiri you can also grab recent APK and sideload it: https://www.apkmirror.com/apk/robert-koch-institut/corona-warn-app/corona-warn-app-1-1-1-release/corona-warn-app-1-1-1-android-apk-download/

Note that it won't update automatically, and as soon as CWA is released in your Play Store country you should uninstall it and install from Play Store, but for now I guess using it like this is better than not using at all.

**Comment by rebelgiri** [comment]
Thank you all for your support, I will close this issue as resolved. I am using APK for now as suggested by @kbobrowski.
@daimpi  Yes I will upvote it.
